tests/FSharp.Compiler.ComponentTests/CompilerService/AsyncMemoize.fs

```diff
@@ -34,6 +34,7 @@ let internal observe (cache: AsyncMemoize<_,_,_>) =
 
 let rec awaitEvents next condition =
     async {
+        do! Async.Sleep 10
```

Comment:
Why is the sleep necessary? And why 10 secs? :)

Reply:
10 ms :), it's arbitrary. Theoretically Async.SwitchToThreadPool should suffice instead. It's here to prevent the possibility of a deadlock when awaiting job events.

Reply:
On second thought, this absolutely should not be required here. I'll look into it again.

Reply:
That was my first thought :) Sure, thanks.